firstvoices/backend/resources/utils/import_export_widgets.py

```diff
@@ -105,13 +105,26 @@ def __init__(self, *args, **kwargs):
 
     def clean(self, value, row=None, **kwargs):
         category_match_pattern = r"category[_2-5]*"
```

Comment:
Just so we don't get confused in future about whether this is matching category names vs the names of category columns, could we please change the name to something like category_column_name_pattern?